src/js/actions/documents.js

```diff
@@ -1003,6 +1004,21 @@ define(function (require, exports) {
         }.bind(this);
         descriptor.addListener("open", _openHandler);
 
+        _closeHandler = function (event) {
+            if (typeof event.documentID === "number") {
```

Comment:
It's almost always better for these handlers to be actions. Is there a good reason for this not to be an action?